fuse/src/mapping.ts

```diff
@@ -209,3 +199,92 @@ export function handleWhitelistedRemoved(event: WhitelistedRemoved): void {
   citizen.save()
 }
 
+function aggregateDailyUbiFromUBIClaimed(event: UBIClaimed, dailyUbi: DailyUBI | null): void {
+  if (dailyUbi.totalUBIDistributed == null) {
+    dailyUbi.totalUBIDistributed = event.params.amount
+  } else {
+    dailyUbi.totalUBIDistributed = dailyUbi.totalUBIDistributed.plus(event.params.amount)
+  }
+
+  if (dailyUbi.totalClaims == null) {
+    dailyUbi.totalClaims = BigInt.fromI32(1)
+  } else {
+    dailyUbi.totalClaims = dailyUbi.totalClaims.plus(BigInt.fromI32(1))
+  }
+
+  let statistics = Statistics.load('statistics')
```

Comment:
get this value from when you update the statistics record in aggregateStatisticsFromUBIClaimed. so we dont load it twice.

Reply:
done